src/useSubscription/models/channel.ts

```diff
@@ -193,6 +193,14 @@ export default class Channel<T extends Action = Action> {
     }
   }
 
+  public refresh(): Promise<void> {
+    this.scope.stop()
```

Comment:
Not even sure it returns a promise, but should this be awaited?

Reply:
I also wondered that but no its a sync operation
https://github.com/vuejs/core/blob/main/packages/reactivity/src/effectScope.ts#L81